### DISCUSSION TITLE
How well has Google Vision worked for people?

### DISCUSSION THREAD

**Discussion by alehel** [SubtitleEdit/subtitleedit#6795] [General]
I currently use Tesseract for creating srt from VOBSUBs from my DVD rips. I was wondering if I should give Google Vision a try, so I tried doing a few Google searches, but wasn't able to find anything about how well it works. I was therefore wondering if anyone could share their experience before I take the time to set this up. Has it performed well, specifically, has it resultet in fewer mistakes than Tesseract 3? I use 3 due to 5 not being able to detect italics, but it leaves me with a lot of errors to correct. Does Google Vision support Italics?

**Comment by niksedk** [comment]
>Does Google Vision support Italics?

Sorry, no. Not as far as I know.
